MANIFEST.in

```diff
@@ -0,0 +1,25 @@
+# Include everything in these directories
+graft sssom
+graft tests
+
+# Don't include anything in these directories
+prune docs
+prune schema
+
+# TODO these should use the tmpfile harness
+prune tests/tmp
+prune tests/data
+
+recursive-include sphinx *.py
+recursive-include sphinx *.rst
```

Comment:
@cthoyt :  sphinx is for documentation in GitHub-pages. Should this be packaged?

Reply:
that's a really good question. I've been doing this for a while but it's not actually used by the code, so maybe we should skip it for now